src/trainer.py

```diff
@@ -1056,6 +1065,7 @@ def _save_checkpoint(self, training_state, phase="pretrain", new_best_macro=Fals
         else:
             best_str = ""
 
+        # Save the overall best model for this target train in terms of macro/micro_avg score.
```

Comment:
Seems misleading, since we'll also run the below line when this checkpoint isn't the best, and best_str == "".